### ISSUE TITLE
[A11Y] [iOS] Unable to navigate cells after button taps

### ISSUE THREAD

**Issue by camelburrito** [ampproject/amphtml#10006]
ENVIRONMENT:
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari

Pre-requisite:
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

STEPS TO REPRODUCE:

Navigate to the URL: https://ampstart.com
Linearly navigate to the “Build with AMP Start” section.
Linearly navigate through the template carousel and move focus on the “previous item in carousel” and "Next item in carousel" buttons and double tap on it.
Try to navigate further.
ACTUAL RESULTS:
Focus is stuck on those buttons when double tapped and does not navigate further.

EXPECTED RESULTS:
User should be able to double tap on either of the buttons and be able to perform respective actions and navigate further.

ATTACHMENT
Please refer to the Video in the below link for reference:

https://drive.google.com/corp/drive/folders/0BwgKOOZP99EuUzVWT25TVzl2S00

Original Issue - https://github.com/ampproject/ampstart/issues/394

**Comment by aghassemi** [comment]
dup of https://github.com/ampproject/amphtml/issues/5487 various attempts have been made in fixing this: https://github.com/ampproject/amphtml/pull/8822 https://github.com/ampproject/amphtml/pull/6333
